config/hardware/axis/XY/TMC/xy_TMC2209.cfg

```diff
@@ -0,0 +1,19 @@
+# XY TMC2209 definition
+[gcode_macro _USER_VARIABLES]
+variable_xytmc_driver: "TMC2209"
+gcode:
+
+
+[tmc2209 stepper_x]
+uart_pin: X_TMCUART
+interpolate: False
+run_current: 0.8
+sense_resistor: 0.110
+stealthchop_threshold: 0
+
+[tmc2209 stepper_y]
+uart_pin: Y_TMCUART
+interpolate: False
+run_current: 0.8
+sense_resistor: 0.110
+stealthchop_threshold: 0
```

Comment:
is here an empty line missing at the end?

Reply:
No, it's here. Just github don't show it. FYI, when it miss an empty newline at the end, Github show a red symbol instead